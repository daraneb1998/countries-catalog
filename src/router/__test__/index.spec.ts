import { describe, it, expect } from "vitest";

import { mount } from "@vue/test-utils";
import router from "..";

describe("router", () => {
  it("renders properly", () => {
    const wrapper = mount(router);
    expect(wrapper.text()).toMatchSnapshot();
  });
});
