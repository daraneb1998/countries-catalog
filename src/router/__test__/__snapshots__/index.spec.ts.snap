// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Pagination > renders properly 1`] = `""`;

exports[`router > renders properly 1`] = `""`;
